resources/styles/components/task-plan/homework/exercise-summary.less

```diff
@@ -48,14 +61,17 @@
     }
     &.total h2 { color: @tutor-quaternary; }
   }
+
   .tutor-added-later {
-    color: @tutor-neutral;
-    width: 200px;
+    color: #999999;
+    width: 232px;
```

Comment:
hm, the `#999999` seems like it might be a useful grey in between `@tutor-neutral` and `@tutor-neutral-light`.  No idea on what to call it though; if you can think of something maybe make that a var in `colors.less`